samtranslator/intrinsics/actions.py

```diff
@@ -505,7 +512,7 @@ def resolve_resource_id_refs(
     def _check_input_value(self, value: Any) -> bool:
         # Value must be an array with *at least* two elements. If not, this is invalid GetAtt syntax. We just pass along
```

Comment:
nit, `*at least* _MIN_NUM_ARGUMENTS elements`

Reply:
updated to "enough"